tfexec/output.go

```diff
@@ -34,23 +39,35 @@ type OutputMeta struct {
 
 // Output represents the terraform output subcommand.
 func (tf *Terraform) Output(ctx context.Context, opts ...OutputOption) (map[string]OutputMeta, error) {
-	outputCmd := tf.outputCmd(ctx, opts...)
```

Comment:
This looks like a reasonable change, but do you mind raising a separate PR for it or explain why does it really need to be part of this PR? 

It would make it much easier to review and ensure that any side effects of that change get the right attention if it was a separate PR.